data/json/items/tool/lighting.json

```
@@ -365,8 +364,7 @@
   {
     "id": "heavy_flashlight",
     "type": "TOOL",
-    "name": "heavy duty flashlight (off)",
-    "name_plural": "heavy duty flashlights (off)",
+    "name": { "str": "heavy duty flashlight (off)", "str_pl": "heavy duty flashlight (off)" },
```

```suggestion
    "name": { "str": "heavy duty flashlight (off)", "str_pl": "heavy duty flashlights (off)" },
```
Oh HOH, this one is on YOU, snipercup. I knew I'd catch you one day. Gotta get up pretty early in the afternoon to slip one past me.

---

Even after testing. thanks